src/Seaborn.jl

```diff
@@ -53,706 +11,115 @@ function __init__()
-* `markers` : boolean, list, or dictionary, optional
-    Object determining how to draw the markers for different levels of the style variable. Setting to True will use default markers, or you can pass a list of markers or a dictionary mapping levels of the style variable to markers. Setting to False will draw marker-less lines. Markers are specified as in matplotlib.
-* `style_order` : list, optional
-    Specified order for appearance of the style variable levels otherwise they are determined from the data. Not relevant when the style variable is numeric.
-* `units` : {long_form_var}
-    Grouping variable identifying sampling units. When used, a separate line will be drawn for each unit with appropriate semantics, but no legend entry will be added. Useful for showing distribution of experimental replicates when exact identities are not needed.
-* `estimator` : name of pandas method or callable or None, optional
-    Method for aggregating across multiple observations of the y variable at the same x level. If None, all observations will be drawn.
-* `ci` : int or “sd” or None, optional
-    Size of the confidence interval to draw when aggregating with an estimator. “sd” means to draw the standard deviation of the data. Setting to None will skip bootstrapping.
-* `n_boot` : int, optional
-    Number of bootstraps to use for computing the confidence interval.
-* `sort` : boolean, optional
-    If True, the data will be sorted by the x and y variables, otherwise lines will connect points in the order they appear in the dataset.
-* `err_style` : “band” or “bars”, optional
-    Whether to draw the confidence intervals with translucent error bands or discrete error bars.
-* `err_band` : dict of keyword arguments
-    Additional paramters to control the aesthetics of the error bars. The kwargs are passed either to ax.fill_between or ax.errorbar, depending on the err_style.
-* `legend` : “brief”, “full”, or False, optional
-    How to draw the legend. If “brief”, numeric hue and size variables will be represented with a sample of evenly spaced values. If “full”, every group will get an entry in the legend. If False, no legend data is added and no legend is drawn.
-* `ax` : matplotlib Axes, optional
-    Axes object to draw the plot onto, otherwise uses the current Axes.
-* `kwargs` : key, value mappings
-    Other keyword arguments are passed down to plt.plot at draw time.
-
-# Returns:	
-
-`ax` : matplotlib Axes
-    Returns the Axes object with the plot drawn onto it.
-"""
-function lineplot(args...; kwargs...)
-    seaborn.lineplot(args...; kwargs...)
+function show(io::IO, ::MIME"text/plain", h::LazyHelp)
+    o = h.o
+    for k in h.keys
+        o = o[k]
+    end
+    if hasproperty(o, "__doc__")
+        print(io, convert(AbstractString, o."__doc__"))
+    else
+        print(io, "no Python docstring found for ", h.k)
```

Comment:
```suggestion
        print(io, "no Python docstring found for ", h.keys)
```
(looks like a typo in PyPlot)